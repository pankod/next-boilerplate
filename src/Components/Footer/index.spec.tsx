// #region Global Imports
import * as React from "react";
import { shallow, mount } from "enzyme";
// #endregion Global Imports

// #region Local Imports
import { Footer } from "@Components";
import { execOnce } from "next-server/dist/lib/utils";
// #endregion Local Imports

describe("Footer", () => {
    it("should render without fail", () => {
        const wrapper = shallow(<Footer />);

        expect(wrapper.find("div.footer").exists()).toBe(true);
    });

    it("should match snapshot", () => {
        const wrapper = mount(<Footer>Test</Footer>);
        expect(wrapper).toMatchSnapshot();
    });
});
