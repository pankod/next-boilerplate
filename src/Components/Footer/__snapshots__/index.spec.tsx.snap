// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<Footer>
  <div
    className="footer"
  >
    Footer
  </div>
</Footer>
`;
